### DISCUSSION TITLE
Add Nuxt guide to project homepage

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#17902] [Contribute to Supabase]
We should add this guide to the project homepage 
https://github.com/nuxt-modules/supabase/tree/main/demo

<img width="1477" alt="screenshot-2023-10-03-at-22 12 18" src="https://github.com/supabase/supabase/assets/105593/69271c49-7795-4dd2-91cf-7f2547dfab27">

**Comment by mirchandani-mohnish** [comment]
Hey, Can I take this up? With a bit of guidance if possible... I have gone through the contributor guidelines and have also setup the local development environment with Supabase. 

and I understand that the issue is up for grabs as written in the guidelines. I just want to know that should I directly open a new issue and fix this and make a pull request once done?

**Comment by mirchandani-mohnish** [comment]
![image](https://github.com/supabase/supabase/assets/80059522/2ea82dde-9a00-4357-a91d-17ffbe4be035)



So two things here 
1. (I apologize for the naive nature ) if we put this link in, the grid above loses its shape ( somewhat)
2. Where can I place the icon for the same?

**Comment by alt-glitch** [comment]
Made a PR here: https://github.com/supabase/supabase/pull/20336
